### DISCUSSION TITLE
Mailscanner, ticket status and duplicated email

### DISCUSSION THREAD

**Discussion by OkayashiM** [YetiForceCompany/YetiForceCRM#16770] [How to] [answered]
I have noticed limitation which I would like to remove (from core code too).
Here we go:
When contact email address is found in users module, scanner action doesnt change ticket status. If I remove user which email corresponding to contact, everything is OK. 
How to fix it?

Okayashi

**Comment by mariuszkrzaczkowski** [comment]
why do you have the same email address in contact/user?

**Comment by OkayashiM** [comment]
As in my scenario I need to grand access for customers to system (based on certain privileges). It may actually happen often that contact become user (employee, customer).

**Reply by mariuszkrzaczkowski** [reply]
I don't know what your business model is
but in my opinion the customer should not have access to the CRM as already the customer portal. It is dangerous to let customers into CRM.
CRM should not be available to the world, only the Portal, this is how it was designed and this is how it is done in most companies.
This is one example of why mail cannot duplicate, the system must be able to tell who is who and adding duplicates in both places defies logic.
Sometimes even two portals are used for customers and partners.

**Comment by OkayashiM** [comment]
Appreciate pointing way around if for some reason this cannot be deployed in the core

Thanks
Okayashi

**Reply by mariuszkrzaczkowski** [reply]
so far I don't see the point of your example, it causes more harm than good, the scanner is just one example where there are problems

**Comment by s2b-git** [comment, marked as answer]
ContactAccount/Vendor etc  = user@domain.tld
User = user+whatever@domain.tld

Above is a solution to rid of above issue.

Best

**Reply by OkayashiM** [reply]
Clever approach. Thanks

Okayashi
